pkgs/applications/graphics/hdr-plus/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub
+, cmake, halide
+, libpng, libjpeg, libtiff, libraw
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hdr-plus-unstable";
+  version = "2020-10-29";
```

Comment:
```suggestion
  pname = "hdr-plus";
  version = "unstable-2020-10-29";
```

Reply:
See the comment earlier that [told me to do it this way](https://github.com/NixOS/nixpkgs/pull/102922#discussion_r525618850)?

Reply:
I am somewhat sure we do it like this.

Reply:
We actually do both and there's a growing number of packages doing it this way. Without an RFC or something to further clarify this, I now agree with the reading of the guidelines that places it in pname, especially since it is also stated that `version` **must** start with a number.

That said, the commit title prolly shouldn't include the "-unstable" bit in the pkg-name, judging by how other package commits handle it. I'll fix the title & force push, without touching this though.

Reply:
Please change it. This only applies to packages with multiple versions.

Reply:
I'm confused, where in the packaging guidelines does it say that? We have multiple packages in Nixpkgs with only 1 packaged version and "unstable" in the pname, some almost as recent as this PR: https://github.com/NixOS/nixpkgs/pull/101384#discussion_r510519152

Reply:
I don't know. We just keep it how it is now.